source_gen/CHANGELOG.md

```diff
@@ -8,13 +10,14 @@
 - Document deduplication behavior for the output of
   `GeneratorForAnnotation.generateForAnnotatedElement`.
 - Support all the glob quotes.
-- Require `analyzer: ^6.9.0`
-- Require Dart 3.6.0
+- Require `analyzer: '>=7.2.0 <8.0.0'`
+- Support the latest `package:dart_style`
```

Comment:
We should mention the new API here

Reply:
I wrote something, but I'm not sure if this is exactly what you expected.